contrib/building.md

```diff
@@ -168,6 +169,14 @@ mklink /j assets "../../../../../data"
 
 Note: If you have cloned the repo previously make sure you remove your `node_modules` directory since it may contain stale dependencies which may cause the build to fail.
 
+### Visual Studio Code setup for C++
+
+If you are using Visual Studio Code as your editor, you can get greatly improved C++ Intellisense by installing the [clangd](https://marketplace.visualstudio.com/items?itemName=llvm-vs-code-extensions.vscode-clangd) extension. This should prompt you to disable the built in C++ extensions Intellisense, but if not you should do this in Settings, searching for `cpp intellisense`.
```

Comment:
why not add this as a suggestion extension in VS Code for this repo

Reply:
I had no idea that was possible! Nice, will update 👍